std_msgs/String emotion
float64 intensity